### PR TITLE
Betatest Option for realistic lighting and atmosphere

### PR DESCRIPTION
**Pull request by nosy-b**
Added scattering light shader and global lighting function.

Remain the rendering tu update but as the lighting is also tested on each tile, we parse tree to update but it would need a renderScene. For now , remember to move mouse to see effect
It will be improved.

### CODE REVIEW

**Comment by nosy-b** [comment]
scattering
A few more optimisations and we will be in the run for the best looking web globe ;)

File: src/Renderer/Shader/CloudsFS.glsl [hunk 1]
```diff
@@ -44,5 +46,9 @@ void main()
 
     gl_FragColor.a = min(time * min( coefDistCam, 1.2) , 1.) * (vUv.y <= 0.75 ? l : (1. - ((vUv.y - 0.75) / 0.25)) * l  );
 
+    if(lightingOn == 1){   // Add lighting
```
Comment (mbredif) [inline src/Renderer/Shader/CloudsFS.glsl:49]: I'd rather see these types of switches as #if tests, could you update the PR accordingly ?

**Comment by tbroyer** [comment]
You may want to squash all your commits into just one or two for a cleaner Git history (just sayin')

UPDATE: oops, too late.

**Comment by nosy-b** [comment]
You're right.. :)

**Pull request merged**